cmake/projects/OpenBLAS/schemes/OpenBLAS.cmake.in

```diff
@@ -49,6 +49,48 @@ else()
   set(lib_type "")
 endif()
 
+set(compiler_arg "@CMAKE_C_COMPILER@")
+
+
+if(CMAKE_CROSSCOMPILING)
+  # Tells OpenBLAS where to find host CC (HOSTCC=)
+  # and not to run tests (CROSS=1)
+  # and not to use fortran (NOFORTRAN=1)
+  # Feel free to implement cross fortran
+
+  # What is HOSTCC?
+  set(HOSTCC gcc)
+  if($ENV{CC})
+    set(HOSTCC $ENV{CC})
+  elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "Darwin")
+    set(HOSTCC "clang")
+  endif()
+ 
+  set(cross_arg HOSTCC=${HOSTCC} CROSS=1 NOFORTRAN=1 ONLY_CBLAS=1)
+
+  # We also need to manually set target when cross compiling
+  if(ANDROID)
+    if(CMAKE_ANDROID_ARCH_ABI STREQUAL "armeabi-v7a")
+      set(target_arg TARGET=ARMV7 BINARY=32 ARM_SOFTFP_ABI=1)
+    elseif(CMAKE_ANDROID_ARCH_ABI STREQUAL "arm64-v8a")
+      set(target_arg TARGET=CORTEXA57 BINARY=64)
+    elseif(CMAKE_ANDROID_ARCH_ABI STREQUAL "x86")
+      set(target_arg TARGET=CORE2 BINARY=32)
+    elseif(CMAKE_ANDROID_ARCH_ABI STREQUAL "x86_64")
+      set(target_arg TARGET=PENRYN BINARY=64)
+    endif()
+  elseif(IOS)
+    set(target_arg TARGET=generic)
+  endif()
+  set(compiler_arg "${compiler_arg} --sysroot @CMAKE_SYSROOT@")
```

Comment:
You should use `hunter_dump_cmake_flags`:
* https://github.com/ruslo/hunter/blob/d41291804cf9ceba09742581029a85773e9f3eee/cmake/modules/hunter_dump_cmake_flags.cmake#L46-L48

It's not only `--sysroot` - there are a lot of other cases.

I personally think it make sense to fix CMakeLists.txt of OpenBLAS then tweaking this scheme.

Reply:
Thanks!
I have CMakeLists working already but it will take a long time to get it merged upstream. 
https://github.com/xianyi/OpenBLAS/pull/1279

Reply:
>  it will take a long time to get it merged upstream

We already using fork:
* https://github.com/hunter-packages/OpenBLAS

Reply:
To clarify: I don't mean CMake code should not be improved upstream, I mean that you can test/merge your changes faster.